logger.go

```diff
@@ -162,6 +162,11 @@ func (sumoLogger *sumoLogger) sendLogs(logs []*sumoLog) error {
   if sumoLogger.gzipCompression {
     request.Header.Add("Content-Encoding", "gzip")
   }
+  if sumoLogger.sourceCategory != "" {
+    request.Header.Add("X-Sumo-Category", sumoLogger.sourceCategory)
+  }
+  request.Header.Add("X-Sumo-Name", sumoLogger.sourceName)
```

Comment:
I know these should never be empty, but would be good to do `!= ""` check on these too.

Reply:
ok. I can add the checking